pkg/registry/core/service/storage/storage.go

```diff
@@ -30,12 +30,12 @@ import (
 	"k8s.io/kubernetes/pkg/registry/core/service"
 )
 
-type REST struct {
+type GenericREST struct {
```

Comment:
why the name change?

Reply:
There are two storage REST types.  One is the raw underlying layer, one is the wrapper.  Rather than use "storage" and "rest", and given that rest is our consistently used name now, used "REST" for the wrapper that is consumed and "GenericREST" for the inner wrapper.

Reply:
:+1: